src/EnergyPlus/HVACInterfaceManager.hh

```diff
@@ -150,7 +149,7 @@ namespace HVACInterfaceManager {
                                 Real64 &MixedOutletTemp // inlet temperature to the common pipe passed in from the capacitance calculation
     );
 
-    void ManageTwoWayCommonPipe(EnergyPlusData &state, int LoopNum, const DataPlant::LoopSideLocation LoopSide, Real64 TankOutletTemp);
+    void ManageTwoWayCommonPipe(EnergyPlusData &state, PlantLocation const &plantLoc, Real64 TankOutletTemp);
```

Comment:
Hmmm, this is one that feels a bit odd.  This is at the plant loop side level, so I imagine for this to work, the plant solver has to just use placeholder values for the branch and comp.  I think this one should go back to just loop num and loop side.

Reply:
I disagree. Just ignore those two fields. Get used to passing `PlantLoc` around instead of its components. Keeping `PlantLoc` here will also enable the follow on pass.